File: app/src/main/java/com/example/myapplication/Other/LeftRightSpacesItemDecoration.java
package com.example.myapplication.Other;
import android.graphics.Rect;
import android.view.View;

import androidx.recyclerview.widget.RecyclerView;
public class LeftRightSpacesItemDecoration extends RecyclerView.ItemDecoration {
    private final int HorizontalSpaceHeight;
    public LeftRightSpacesItemDecoration(int SpaceLenght) {
        this.HorizontalSpaceHeight = SpaceLenght;
    }

    @Override
    public void getItemOffsets(Rect outRect, View view, RecyclerView parent,
                               RecyclerView.State state) {
        outRect.left = HorizontalSpaceHeight;
    }
}

File: app/src/main/java/com/example/myapplication/Other/LeftRightSpacesItemDecoration.kt
package com.example.myapplication.Other

import android.graphics.Rect
import android.view.View
import com.google.gson.JsonDeserializer
import kotlin.Throws
import com.google.gson.JsonParseException
import com.google.gson.JsonElement
import com.google.gson.JsonDeserializationContext
import androidx.recyclerview.widget.RecyclerView.ItemDecoration
import androidx.recyclerview.widget.RecyclerView

class LeftRightSpacesItemDecoration(private val HorizontalSpaceHeight: Int) : ItemDecoration() {
    override fun getItemOffsets(
        outRect: Rect, view: View, parent: RecyclerView,
        state: RecyclerView.State
    ) {
        outRect.left = HorizontalSpaceHeight
    }
}